astropy/io/misc/tests/test_hdf5.py

```diff
@@ -566,6 +566,15 @@ def test_read_h5py_objects(tmpdir):
 
     f.close()         # don't raise an error in 'test --open-files'
 
+@pytest.mark.skipif('not HAS_H5PY')
+def test_write_to_Hdf5(tmpdir):
+    test_file = str(tmpdir.join('test.hdf5'))
+    t1 = Table()
+    t1['x']=['a', 'b', 'c']
```

Comment:
Can you add at least one column with a different type, to check that the type detection is correct ?

Reply:
Please also include non-ASCII characters to make sure that this works fine.